data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
 		"drag" 16.1
 		"heat dissipation" .4
 		"fuel capacity" 700
-		"cargo space" 530
-		"outfit space" 740
-		"weapon capacity" 300
-		"engine capacity" 180
+		"cargo space" 240
+		"outfit space" 720
+		"weapon capacity" 280
```

Comment:
I approve of the weapon capacity changes. The Bactrian *isn't* a "capital ship." It's a converted civilian colony transporter, more or less. It still has more hardpoints than most warships, so you'll be encouraged to fill them up with smaller, more numerous weapons. Six heavy laser or quad blaster turrets is still nothing to sneeze at, and there will still be room to spare.